Week2/Sorting/SortAndSearch.java

```diff
@@ -0,0 +1,91 @@
+import java.util.*;
+
+
+
+public class SortAndSearch {
+    static List<Number> numbers = new ArrayList<>();
+    static Set<Number> noDuplicate = new HashSet<>();
+
+
+
+    public List<Number> removeDuplicates(List<Number> numbers) {
+        List<Number> originalList = new ArrayList<>();
+        noDuplicate.addAll(numbers);
+        originalList.addAll(noDuplicate);

```

Comment:
Please correct return type for removing unnecessary code

Reply:
Fixed.